### PR TITLE
Added Cloudmersive Currency Connector

### PR DESCRIPTION
**Pull request by Cloudmersive**
---
Please check the following conditions for your PR.

- [ X ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ X ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: certified-connectors/Cloudmersive Currency/Readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+## Cloudmersive Video and Media
```
Comment (sriyen-msft) [inline certified-connectors/Cloudmersive Currency/Readme.md:2]: @Cloudmersive The title of your connector in "Cloudmersive Currency", It is a typo?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Cloudmersive Currency/Readme.md (hunk 1 above)
Comment (Cloudmersive) [reply certified-connectors/Cloudmersive Currency/Readme.md:2]: @sriyen-msft Thanks for the feedback, corrected this in latest commit

Comment (dougelliott) [reply certified-connectors/Cloudmersive Currency/Readme.md:2]: @Cloudmersive It looks like the readme that's in `Cloudmersive Currency` still says 'Cloudmersive Video and Media', but now the readme in `Cloudmersive Video and Media` says 'Cloudmersive Currency'

Comment (Cloudmersive) [reply certified-connectors/Cloudmersive Currency/Readme.md]: @dougelliott @sriyen-msft Thanks for the feedback.  New corrections were added to address this.

**Comment by Cloudmersive** [comment]
The requested change was made, just wanted to check in on the review

**Review by dougelliott — APPROVED** [review]

File: certified-connectors/Cloudmersive Currency/Readme.md (hunk 1 above)
Comment (dougelliott) [reply certified-connectors/Cloudmersive Currency/Readme.md]: Thanks!

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**